rest/data/host.go

```diff
@@ -239,6 +239,15 @@ func makeSpawnOptions(options *restmodel.HostRequestOptions, user *user.DBUser)
 	if keyVal == "" {
 		return nil, errors.Errorf("public key '%s' cannot have an empty value", options.KeyName)
 	}
+
+	if options.NoExpiration {
+		if options.SleepScheduleOptions.IsZero() {
+			options.SleepScheduleOptions = host.GetDefaultSleepSchedule(user.Settings.Timezone)
+		} else {
```

Comment:
Should we be hitting `SetDefaultTimeZone` on L247 regardless or do we need this else block?

Reply:
It doesn't really make a difference since `GetDefaultSleepSchedule` already calls `SetDefaultTimeZone`, but yeah, I can remove the else block to keep things simple.